apps/core/client/revalidate-target.ts

```diff
@@ -0,0 +1,3 @@
+export const revalidate = process.env.NEXT_PUBLIC_DEFAULT_REVALIDATE_TARGET
+  ? Number(process.env.NEXT_PUBLIC_DEFAULT_REVALIDATE_TARGET)
+  : undefined;
```

Comment:
If we default to 3600 in `.env.example`, should we also default to 3600 here?

```suggestion
  : 3600;
```

Reply:
Good catch, I think that makes sense.